source/uwp/winui3/AdaptiveCardsObjectModel/AdaptiveCardsObjectModel.vcxproj

```diff
@@ -0,0 +1,333 @@
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveTextInputParser.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveTextRun.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveTimeInputParser.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveToggleInputParser.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveToggleVisibilityActionParser.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveToggleVisibilityTarget.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveToggleVisibilityAction.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveTokenExchangeResource.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveUnsupportedAction.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveUnsupportedElement.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveWarning.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\CustomActionWrapper.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\CustomElementWrapper.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveCard.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveChoiceInput.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveChoiceSetInput.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveColumn.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveColumnSet.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveContainer.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveFact.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveFactSet.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveImage.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveImageSet.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveDateInput.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveNumberInput.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveTextInput.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveTimeInput.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveToggleInput.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveOpenUrlAction.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveShowCardAction.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveSubmitAction.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\AdaptiveTextBlock.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\pch.h" />
+    <ClInclude Include="..\..\sharedObjectModel\lib\ObjectModelUtil.h" />
+    <ClCompile Include="..\..\sharedObjectModel\lib\AdaptiveBackgroundImage.cpp" />
+    <ClCompile Include="..\..\sharedObjectModel\lib\AdaptiveMedia.cpp">
+      <PreprocessToFile Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">false</PreprocessToFile>
```

Comment:
This can be shortened to 
<PreprocessToFile>false</PreprocessToFile>

I don't think in any configuration the property is set to true

Reply:
also, why does this file requires this property?

Reply:
Don't know why Visual Studio stuck this in there, but as I move these to a SharedProject it's going away.